Demo_Project/src/GlobalData/user.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+class USER extends React.Component<RouteComponentProps<{}>> {
```

Comment:
1. class name should be in PascalCase,
2. why this component doesn't have `render` function?
3. if you don't want any React.Component life cycle functions, then this class shouldn't be extending `React.Component` because this will add extra unnecessary jam to the code

Reply:
i remove it because i don't need it any more